packages/xdl/package.json

```
@@ -29,22 +26,22 @@
   },
   "homepage": "https://github.com/expo/expo-cli/tree/master/packages/xdl#readme",
   "dependencies": {
+    "@expo/babel-preset-cli": "^0.1.1",
```

As far as I remember we wanted to keep strict versions (same applies to few dependencies below), so:
```suggestion
    "@expo/babel-preset-cli": "0.1.1",
```

---

Using exact versions is not necessary for `devDependencies`, since those are only used for development, inside the `expo-cli` repository, which has a `yarn.lock` file. But, looks like this dependency got added as a dependency, when it should have been a devDependency. I'll fix that!

---

Uh, I didn't notice what dependency it is, but for example `@expo/json-file`, `@expo/osascript`  and `@expo/config` still don't have strict versions :)

---

Yup, those are fixed now. :+1: